src/models/Settings.js

```diff
@@ -105,6 +105,13 @@ export default Model.extend({
       value: 'SECONDARY',
     },
 
+    // Device Enrollment Remediation
```

Comment:
I doubt we need to change Settings.js from V1 code base.. All of your changes should go to V2 only..

Device enrollment object should be available in appState.. @jhe-okta has added it already..
https://github.com/okta/okta-signin-widget/blob/master/src/v2/ion/responseTransformer.js#L177

Reply:
Right. I just tested locally and it works without adding these in Setting.js. Will remove in next commit.